sources/packages/web/src/components/generic/Separator.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <span class="mx-1 brand-gray-text">|</span>
```

Comment:
can't we use the style itself instead of `|`. maybe `.vertical-divider`

Reply:
@ann-aot can you share the benefits of using the `.vertical-divider` instead of the "|"?
Figma is also using the "|" and it seems that is just to provide the separation between "some text | some text".

Reply:
@andrewsignori-aot Normal dividers are created by borders CSS, with which we can easily add other properties like border-stye (like dashed, rigid etc...), width, height color, etc...  I believe if we use `|` we won't be able to to achieve all the border right